receiver/zipkinreceiver/README.md

```diff
@@ -28,14 +28,11 @@ receivers:
 
 The following settings are configurable:
```

Comment:
If you don't mind, since you're updating this, I think adding documentation for the `parse_string_tags` setting would go well with this change.

Reply:
Thanks, I forgot about this one.